docs/tutorial/index.html

```diff
@@ -17,6 +17,194 @@
+<pre><code>$ ./test --help
+TEST(1)                           Test Manual                          TEST(1)
+
+
+
+NAME
+       test - run tests for my_project
+
+SYNOPSIS
+       test [COMMAND] …
+
+DESCRIPTION
+
+...
+
+COMMANDS
+       approve [--filter-substring=SUBSTRING] [OPTION]…
+           approve new test output
+
+       run [OPTION]…
+           run the tests
+
+       status [OPTION]…
+           show test status
+</code></pre>
+<p>Let’s run our test suite with <code>./test run</code> or just <code>./test</code>:</p>
+<pre><code>$ ./test
+Legend:
+• [PASS]: a successful test that was expected to succeed (good);
+• [FAIL]: a failing test that was expected to succeed (needs fixing);
+• [XFAIL]: a failing test that was expected to fail (tolerated failure);
+• [XPASS]: a successful test that was expected to fail (progress?).
+• [MISS]: a test that never ran;
+• [SKIP]: a test that is always skipped but kept around for some reason;
+• [xxxx*]: a new test for which there&#39;s no expected output yet.
+  In this case, you should review the test output and run the &#39;approve&#39;
+  subcommand once you&#39;re satisfied with the output.
+[PASS]  5d41402abc4b hello
+• Path to captured log: _build/testo/status/my_project/5d41402abc4b/log
+1/1 selected test:
+  1 successful (1 pass, 0 xfail)
+  0 unsuccessful (0 fail, 0 xpass)
+overall status: success</code></pre>
+<p>The script <a href="https://github.com/semgrep/testo/blob/main/docsrc/tutorial/run-tutorial">run-tutorial</a> runs all the steps above to create a sample <code>my_test</code> project.</p>
+<p>Congratulations, the “hello” test passed!</p>
+<p>… but did it, though? Did it output <code>hello!</code> like it was supposed to? The output from <code>./test</code> gives us the path to the captured log:</p>
+<pre><code>$ cat _build/testo/status/my_project/5d41402abc4b/log
+hello!</code></pre>
+<p>The test was successful because it didn’t raised any exception, not because it printed <code>hello!</code> correctly.</p>
+<h3 id="make-the-test-check-its-output">Make the test check its output</h3>
+<p>To enforce that our test prints what it’s supposed to, we’re going to specify that we want to capture the standard output (stdout) produced by the test and compare it against a reference. If the output of the test changes in the future, it will be detected and reported as a test failure.</p>
+<p>First, let’s modify the “hello” test to check stdout by specifying the <code>checked_output</code> option:</p>
+<pre><code>let test_hello =
+  Testo.create &quot;hello&quot;
+    ~checked_output:(Testo.stdout ())
+    (fun () -&gt; print_endline &quot;hello!&quot;)
+</code></pre>
+<p>Running <code>./test</code> with the updated code reports a failure and tells us that something’s missing:</p>
+<pre><code>...
+┌──────────────────────────────────────────────────────────────────────────────┐
+│ [PASS*] 5d41402abc4b hello                                                   │
```

Comment:
maybe a good opportunity here to explain the PASS*.
You have it explained in the legend before, but it's easy to miss. Maybe you can repeat the legend part here, just
the relevant part like [xxx*] ...